### DISCUSSION TITLE
About 'Open With' list

### DISCUSSION THREAD

**Discussion by kaikaibenkai** [tauri-apps/tauri#7337] [Q&A] [answered]
How can I make my application appear in the list?

For example, I created a markdown reader, and when users choose to open `.md` files, I hope my app can appear in the list.

**Comment by FabianLars** [comment, marked as answer]
This is currently not possible but tracked here: https://github.com/tauri-apps/tauri/issues/3736 (please don't add a +1 comment, just add a thumbs up emoji or something)
